src/components/community/DesktopLeftNav.jsx

```
@@ -1,19 +1,25 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import { useLocation, useNavigate } from 'react-router-dom';
 import { useAuth } from '../../contexts/auth-context';
+import { db } from '@/lib/firebase';
+import { collection, onSnapshot, query, where } from 'firebase/firestore';
```

ESLint

[error] 'query' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)